engine/modelc/src/java/com/dynamo/bob/pipeline/ModelImporter.java

```diff
@@ -471,6 +472,18 @@ public static void main(String[] args) throws IOException {
             return;
         }
 
+        // TODO: Setup a java test suite for the model importer
+        int i = 0;
+        for (String arg : args)
```

Comment:
Why not use a for(int i=0; i < args.size(), i++) setup if you need the i in the body?